src/io/github/sspanak/tt9/preferences/helpers/Hotkeys.java

```diff
@@ -117,7 +117,7 @@ private void generateList() {
 
 		addIfDeviceHasKey(KeyEvent.KEYCODE_BACK, R.string.key_back, false);
 		addIfDeviceHasKey(KeyEvent.KEYCODE_CLEAR, R.string.key_clear, false);
-		addIfDeviceHasKey(KeyEvent.KEYCODE_DEL, R.string.key_delete, false);
+		add(KeyEvent.KEYCODE_DEL, R.string.key_delete, false);
```

Comment:
@KaniJonas, I am not so keen on on merging this "temporary fix". It will make the option appear on phones that don't have the key.

Would it be possible for you to find another solution? I would even agree modifying `addIfDeviceHasKey()` to detect the Kyocera using `Build.MODEL`, `Build.MANUFACTURER` or some other method, and always returning `true` for that key. But whatever you choose, please, let's do it in a separate pull request.